Formula/b/bash-completion@2.rb

```diff
@@ -40,7 +40,7 @@ def install
       s.gsub! "(/etc/bash_completion.d)", "(#{etc}/bash_completion.d)"
     end
 
-    system "autoreconf", "-i" if build.head?
+    system "autoreconf", "--force", "--install", "--verbose" if build.head?
     system "./configure", "--prefix=#{prefix}"
```

Comment:
Why not this one?

Reply:
yeah, technically we can, I will update.